frontend/app/assets/css/main.css

```diff
@@ -10,7 +10,7 @@
 
 @theme static {
   --font-sans: 'Montserrat', sans-serif;
-  --font-pixelify: Pixelify Sans;
+  --font-pixelify: Pixelify Sans Hack4Krak;
```

Comment:
Write short comment saying that it is Pixelify Sans with slilghty modified fonts